### DISCUSSION TITLE
[Feature Request] Enable pinning private repositories in my profile

### DISCUSSION THREAD

**Discussion by aravind-craft** [community/community#74784] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'd like to remove all public popular repositories and instead pin my chosen 6 private repositories in the profile page.
Right now its an additional click (or two) to navigate to a private repo that I use often.

**Comment by eymeen** [comment]
@aravind-murthy as I understood from your question, you want to pin 5 private repositories on your public profile, this is not possible because pinning is only for public repositories. When you try to pin, you'll see a message saying `Select up to six **public** repositories or gists you'd like to show to anyone.`

But you can show them on your activity if you want, you're activity is all private

**Comment by ivandda** [comment]
Hi, unfortunately, you can only pin public repos. So you have to make them public to pin them.

**Comment by aravind-craft** [comment]
Yes i know I'm sorry.

This is a feature request to allow private repositories to be pinned.

**Reply by eymeen** [reply]
write that in the title

**Reply by aravind-craft** [reply]
does it look better now?

**Reply by ivandda** [reply]
yes

**Reply by eymeen** [reply]
@aravind-murthy it does

**Comment by vdevcode** [comment]
Yes i know

**Comment by Sintayew4** [comment]
#74784

**Comment by kjjjjjjjjjjjj** [comment]
....
